docs/usage.md

```diff
@@ -223,6 +223,19 @@ const sequelize = new Sequelize('database', 'username', 'password', {
 })
 ```
 
+### ORACLE
```

Comment:
Seems the other database names are not uppercased. `ORACLE` should be `Oracle` or `Oracle Database`.